api/client/diff.go

```diff
@@ -34,7 +34,7 @@ func (cli *DockerCli) CmdDiff(args ...string) error {
 		var kind string
 		switch change.Kind {
 		case archive.ChangeModify:
-			kind = "C"
+			kind = "M"
```

Comment:
OK, since i am a newer to docker, so it seems hard for me to modify all of this, but i will report an issue for this problem. And here i will cancel the change of character "C".